qiita_pet/handlers/study_handlers/listing_handlers.py

```diff
@@ -42,13 +43,40 @@ def _get_shared_links_for_study(study):
     return ", ".join(shared)
 
 
-def _build_study_info(user, results=None):
-    """builds list of dicts for studies table, with all html formatted"""
+def _build_study_info(user, study_proc=None, proc_samples=None):
```

Comment:
refactored and now renders the gnarly stuff client-side usingt he datatables module itself.